app/src/components/domain/GlobalMap/index.tsx

```diff
@@ -0,0 +1,238 @@
+    );
+
+    const adminZeroLineLayerOptions = useMemo<Omit<LineLayer, 'id'>>(
+        () => ({
+            type: 'line',
+            layout: {
+                visibility: 'visible',
+            },
+        }),
+        [],
+    );
+
+    const { currentLanguage } = useContext(LanguageContext);
+
+    const adminLabelLayerOptions : Omit<SymbolLayer, 'id'> = useMemo(
+        () => {
+            // ar, es, fr
+            let label: string;
+            if (currentLanguage === 'es') {
+                label = 'name_es';
+            } else if (currentLanguage === 'ar') {
+                label = 'name_ar';
+            } else if (currentLanguage === 'fr') {
+                label = 'name_fr';
+            } else {
+                label = 'name';
+            }
+
+            return {
+                type: 'symbol',
+                layout: {
+                    'text-field': ['get', label],
+                    visibility: 'visible',
+                },
+            };
+        },
+        [currentLanguage],
+    );
+
+    return (
+        <BaseMap
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...baseMapProps}
+            withoutLabel
+            baseLayers={(
+                <>
+                    <MapLayer
+                        layerKey="admin-0"
+                        layerOptions={adminZeroBaseLayerOptions}
+                        onMouseEnter={handleFeatureMouseEnter}
+                        onMouseLeave={handleFeatureMouseLeave}
+                    />
+                    <MapLayer
+                        layerKey="admin-0-label"
+                        layerOptions={adminLabelLayerOptions}
+                    />
+                    <MapLayer
+                        layerKey="admin-0-label-non-independent"
+                        layerOptions={adminLabelLayerOptions}
+                    />
+                    <MapLayer
+                        layerKey="admin-0-label-priority"
+                        layerOptions={adminLabelLayerOptions}
+                    />
+                    <MapLayer
+                        layerKey="admin-0-boundary-mask"
+                        layerOptions={adminZeroLineLayerOptions}
+                    />
+                    <MapLayer
+                        layerKey="admin-0-boundary-disputed"
+                        layerOptions={adminZeroLineLayerOptions}
+                    />
+                    {(onHover || onClick) && (
+                        <MapLayer
+                            layerKey="admin-0-highlight"
+                            layerOptions={adminZeroHighlightLayerOptions}
+                            onClick={onClick ? handleClick : undefined}
+                        />
+                    )}
+                    {/*
+                    <MapLayer
+                        layerKey="admin-0-boundary"
+                        layerOptions={adminZeroLineLayerOptions}
+                    />
+                    */}
```

Comment:
Please clarify why this code is commented out or consider removing it.

Reply:
This has been removed because it's implemented on GlobalMap instead.

Reply:
We cannot override layout for "admin-0-boundary" else all of the hidden layers will also be visible.

@frozenhelium do you know why this happens?